code/game/turfs/turf.dm

```diff
@@ -104,6 +104,9 @@ GLOBAL_LIST_EMPTY(station_turfs)
 	/// This would either be expensive, or impossible to manage. Let's just avoid it yes?
 	/// Never directly access this, use get_explosive_block() instead
 	var/inherent_explosive_resistance = -1
+	/// The biome this turf was generated in, if any. null by default.
+	var/datum/biome/biome = null
```

Comment:
I made it a direct reference, because I figured it likely would be faster than having to lookup by type, and likely take less space in memory, considering how many turfs we're talking about. I don't actually know if there's a size difference between references and typepaths, either. I'm willing to change this if requested to change it.

Reply:
Removed it.